.github/workflows/ci.yml

```diff
@@ -9,19 +9,19 @@ on:
 jobs:
   pod-lint:
     name: Pod Lint
-    runs-on: macos-13
+    runs-on: macos-14
```

Comment:
migrating to macOS 14 [which is now ARM](https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/) while we're here